src/OnchainKitProvider.tsx

```diff
@@ -10,16 +10,24 @@ export const OnchainKitContext = createContext<OnchainKitContextType>(ONCHAIN_KI
  */
 export function OnchainKitProvider({
   address,
+  apiKey,
   chain,
   children,
   schemaId,
 }: OnchainKitProviderProps) {
   if (schemaId && !checkHashLength(schemaId, 64)) {
     throw Error('EAS schemaId must be 64 characters prefixed with "0x"');
   }
+  if (!apiKey) {
+    throw Error(
```

Comment:
is it? I don't think so!!!

ApiKey is required only in a few utilities, and for those we should alert missing the apiKey.

OnchainKit will always be bigger than what the Api offers.

Reply:
removed mandatory api key